core/src/main/java/hudson/model/View.java

```diff
@@ -813,37 +826,41 @@ public AsynchPeople(View parent) {
             int itemCount = 0;
             for (Item item : items) {
                 for (Job<?,?> job : item.getAllJobs()) {
-                    if (job instanceof AbstractProject) {
-                        AbstractProject<?,?> p = (AbstractProject) job;
-                        RunList<? extends AbstractBuild<?,?>> builds = p.getBuilds();
+                    if (job instanceof SCMTriggerItem) {
+                        RunList<? extends Run<?,?>> runs = job.getBuilds();
                         int buildCount = 0;
-                        for (AbstractBuild<?,?> build : builds) {
+                        for (Run<?,?> r : runs) {
                             if (canceled()) {
                                 return;
                             }
-                            for (ChangeLogSet.Entry entry : build.getChangeSet()) {
-                                User user = entry.getAuthor();
-                                UserInfo info = users.get(user);
-                                if (info == null) {
-                                    UserInfo userInfo = new UserInfo(user, p, build.getTimestamp());
-                                    userInfo.avatar = UserAvatarResolver.resolveOrNull(user, iconSize);
-                                    synchronized (this) {
-                                        users.put(user, userInfo);
-                                        modified.add(user);
-                                    }
-                                } else if (info.getLastChange().before(build.getTimestamp())) {
-                                    synchronized (this) {
-                                        info.project = p;
-                                        info.lastChange = build.getTimestamp();
-                                        modified.add(user);
+                            if (r instanceof RunWithSCM) {
+                                RunWithSCM runWithSCM = (RunWithSCM) r;
+                                for (ChangeLogSet<? extends ChangeLogSet.Entry> c : runWithSCM.getChangeSets()) {
```

Comment:
Tip: to minimize diff hunk size for reviewers (and thus reduce chance of merge conflicts, `git blame` breakage, etc. etc.), you can keep the original indentation without violating code style:

```java
for (Run<?,?> r : runs) {
    if (canceled()) {
        return;
    }
    if (!(r instanceof RunWithSCM)) {
        continue;
    }
    for (ChangeLogSet.Entry entry : ((RunWithSCM) r).getChangeSet()) {
        // …rest untouched
```